bookkeeper-server/src/main/java/org/apache/bookkeeper/http/BKHttpServiceProvider.java

```diff
@@ -196,7 +196,11 @@ public HttpService provideReadLedgerEntryService() {
      */
     @Override
     public HttpService provideListBookieInfoService() {
```

Comment:
There's an inconsistency in the scope of the data returned from this two endpoints. Bookie info returns the bookie info for the whole cluster, while list disk files returns the files for only one node.

I would suggest putting bookie info under /api/v1/bookkeeper/